boards/ikea-tradfri/include/board.h

```diff
@@ -31,10 +31,14 @@ extern "C" {
 /**
  * @name    Xtimer configuration
  *
- * The timer runs at 250 KHz to increase accuracy.
+ * The timer runs at 250 KHz to increase accuracy or 32768 Hz for LETIMER
```

Comment:
Here and in other places:

```suggestion
 * The timer runs at 250 KHz to increase accuracy or 32768 Hz for LETIMER.
```

Reply:
And while you're at it: I also realize that `250 KHz` is incorrect and should be `250 kHz`. 

I probably made that mistake, so feel free to leave that one for me in a future PR :-)